invenio_cli/commands/services_health.py

```diff
@@ -28,17 +29,16 @@ class ServicesHealthCommands(object):
     @classmethod
     def search_healthcheck(cls, *args, **kwargs):
         """Open/Elasticsearch healthcheck."""
-        verbose = kwargs["verbose"]
-
+        host = kwargs["search_host"]
+        port = kwargs["search_port"]
         return run_cmd(
-            ["curl", "-f", "localhost:9200/_cluster/health?wait_for_status=yellow"]
+            ["curl", "-f", f"{host}:{port}/_cluster/health?wait_for_status=yellow"]
```

Comment:
Was this part of allowing a local instance and the tests to run at the same time? I guess it could be tolerant to multiple document engines running on different ports e.g., work on different projects and this makes it easier to switch between them. 

The host part I am less convinced by, but we are letting people pass a host and port for the server already, so c'est-la-vie. In general, I think we should steer people away from using invenio-cli to run elaborate multi-machine setup and a good way is to not let them. Power users can usually figure it out anyway (have the skill to create workaround scripts). For newcomers it's another way to get confused. But here it's done tastefully and there is prior art, so that's fine.

Reply:
yes this changes makes it possible to run multiple instances at the same time. the thing is i couldn't make it work without that change. using only docker-compose files is not enough because the healthcheck in invenio-cli blocks the `invenio-cli run` command.

but those `kwargs["search_host"]` values have default values so the current behavior still works!